deploy/chart/templates/0000_50_olm_15-packageserver.clusterserviceversion.yaml

```diff
@@ -0,0 +1,126 @@
+{{ if eq .Values.installType "ocp" }}
```

Comment:
`_packageserver.clusterserviceversion.yaml` is a named partial template that you can [use the `include` keyword to pull in](https://docs.helm.sh/chart_template_guide/#named-templates) (then it will always be up to date).

Reply:
The reason I didn't include the file is the syntax is slightly different. This file starts off with just `apiVersion...` whereas the _packageserver.clusterserviceversion.yaml file starts off with a list item:
`- apiVersion...`
If you can figure out how to make it come out right with those differences let me know.

Reply:
yup, you are totally right - If we were going to fix that I think we would need to remove the list item operator `-` from the sub template and add it to the olm-operators configmap template.

Reply:
I actually tried what you said before and wasn't able to get it to work FWIW.